README.zh-CN.md

```diff
@@ -18,92 +18,94 @@
 <span id="what-is"></span>
 ## Hydra Lab是什么？
```

Comment:
Maybe unify the usage of blank between Chinese words and English phrases?
Currently usage involves some blanks, but not for every single place.

Reply:
Done, thanks.